assets/i18n/en.json

```diff
@@ -44,6 +44,8 @@
     "I18N_ACTION_CREATE_LESSON": "Create your own Lesson",
     "I18N_CARD_HEIGHT_LIMIT_WARNING_MESSAGE": "This card is quite long, and students might lose interest. Consider shortening it, or breaking it into two cards.",
     "I18N_CLASSROOM_PAGE_HEADING": "The Oppia Classroom Page",
+    "I18N_CLASSROOM_PAGE_SEARCH_BAR_HEADING": "Explore More Lessons:",
```

Comment:
Let's drop the colon at the end, and make the "Made by the Community" text a little smaller.

Reply:
Done